reactor-core/src/main/java/reactor/core/publisher/ParallelFlux.java

```diff
@@ -1052,25 +1025,6 @@ public final Disposable subscribe(
 		return this.subscribe(onNext, onError, onComplete, onSubscribe, null);
 	}
 
-	/**
-	 * Subscribes to this {@link ParallelFlux} by providing an onNext, onError and
-	 * onComplete callback as well as an initial {@link Context}, then trigger the execution chain for all
-	 * 'rails'.
-	 *
-	 * @param onNext consumer of onNext signals
-	 * @param onError consumer of error signal
-	 * @param onComplete callback on completion signal
-	 * @param initialContext {@link Context} for the rails
-	 */
-	@Deprecated
```

Comment:
actually, I don't think this one should be removed :( It looks like it was deprecated by mistake, because ultimately the one we'd want removed is the one with a `Consumer<Subscription>` (see the TODO on `subscribe` variant with such a `Consumer` below)

Reply:
please revert the removal of that method, and delete the `@Deprecated` instead 🙇